src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+iexToken=${IEX_TOKEN}
```

Comment:
try to move this into the yml to avoid multiple config files

Reply:
I tried adding it to the YML first, but for some reason it doesn't read the environment variable. It works if I add it to application.properties, so I created that file.

Reply:
For some reason, it wasn't working before when I put it inside the YML. So, I added it to application.properties. But I re-checked now and it works inside the YML as well, so I'll make that change.

Reply:
I've made the changes.